### DISCUSSION TITLE
Support the Existing Policies and Alert

### DISCUSSION THREAD

**Discussion by sathish046** [newrelic/terraform-provider-newrelic#1759] [Q&A]
Hello Team,

Is it possible to support the terraform in existing older policies and Alerts which had been created via REST API?

Regards,
Sathish

**Comment by kidk** [comment]
Hi @sathish046 

You should be able to import existing alerts and policies from New Relic using the terraform import command. It will work even if the alerts were created through the REST API. 

For example: https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/alert_condition#import

Hope this helps
